RobotCode/snobot2018/trajectory_config/StartToScale/CenterToLeftScale.yml

```diff
@@ -1,55 +0,0 @@
-config: !!com.team254.lib.trajectory.gen.TrajectoryGenerator$Config
```

Comment:
Here is the trajectory stuff.  Basically it looks like this was the update to split "start to switch" into two parts, the "go to the switch" and the "if we are on the side, we probably don't want to monopolize the center of the switch, use the side" paths.  Also includes the simplification of "drawing the path to drive" vs "use two points and hope that math works" paths.